lib/workers/branch/automerge.js

```diff
@@ -8,13 +8,22 @@ async function tryBranchAutomerge(config) {
     return 'no automerge';
   }
   const existingPr = await platform.getBranchPr(config.branchName);
-  if (existingPr) {
-    return 'automerge aborted - PR exists';
-  }
+
   const branchStatus = await platform.getBranchStatus(
     config.branchName,
     config.requiredStatusChecks
   );
+
+  if (existingPr) {
+    if (branchStatus === 'success') {
+      const commits = await platform.getCommitMessages();
```

Comment:
No, this just get the last 10 commits for the entire repo. Use `existingPr.canRebase` instead. If it's true then it means there's only one 1 commit and safe to merge.